fast64_internal/oot/oot_actor.py

```diff
@@ -1,47 +1,401 @@
+	propAnnotations['actorKey'] = bpy.props.StringProperty(name='Actor Key', default='0000')
+	propAnnotations['actorParam'] = bpy.props.StringProperty(name = 'Actor Parameter', default = '0x0000',
+		get=lambda self: getValues(self, self.actorID, 'actorParam', 'Params', None),
+		set=lambda self, value: setValues(self, value, 'Params', 'actorID'))
+	propAnnotations['actorParamCustom'] = bpy.props.StringProperty(name = 'Actor Parameter', default = '0x0000')
+
+	# Rotations
+	propAnnotations['rotOverride'] =  bpy.props.BoolProperty(name = 'Rot Override', default = False)
+	propAnnotations['rotOverrideX'] =  bpy.props.StringProperty(name = 'Rot X', default = '0x0',
+		get=lambda self: getValues(self, self.actorID, None, 'XRot', None),
+		set=lambda self, value: setValues(self, value, 'XRot', 'actorID'))
+	propAnnotations['rotOverrideY'] =  bpy.props.StringProperty(name = 'Rot Y', default = '0x0',
+		get=lambda self: getValues(self, self.actorID, None, 'YRot', None),
+		set=lambda self, value: setValues(self, value, 'YRot', 'actorID'))
+	propAnnotations['rotOverrideZ'] =  bpy.props.StringProperty(name = 'Rot Z', default = '0x0',
+		get=lambda self: getValues(self, self.actorID, None, 'ZRot', None),
+		set=lambda self, value: setValues(self, value, 'ZRot', 'actorID'))
+	propAnnotations['rotOverrideCustom'] = bpy.props.BoolProperty(name = 'Override Rotation', default = False)
+	propAnnotations['rotOverrideXCustom'] = bpy.props.StringProperty(name = 'Rot X', default = '0x0')
+	propAnnotations['rotOverrideYCustom'] = bpy.props.StringProperty(name = 'Rot Y', default = '0x0')
+	propAnnotations['rotOverrideZCustom'] = bpy.props.StringProperty(name = 'Rot Z', default = '0x0')
+
+	# We have to use a bool to know what's needed to be exported
+	propAnnotations['XRotBool'] = bpy.props.BoolProperty(default=False)
+	propAnnotations['YRotBool'] = bpy.props.BoolProperty(default=False)
+	propAnnotations['ZRotBool'] = bpy.props.BoolProperty(default=False)
+
+	# Transition Actors
+	propAnnotations['transActorID'] = bpy.props.EnumProperty(name='Transition Actor ID', items=ootEnumTransitionActorID)
+	propAnnotations['transActorIDCustom'] = bpy.props.StringProperty(name='Actor Key', default='ACTOR_CUSTOM')
+	propAnnotations['transActorKey'] = bpy.props.StringProperty(name='Transition Actor ID', default='0009')
+	propAnnotations['transActorParam'] = bpy.props.StringProperty(name = 'Actor Parameter', default = '0x0000',
+		get=lambda self: getValues(self, self.transActorID, 'actorParam', 'Params', 'transActorParam'),
+		set=lambda self, value: setValues(self, value, 'Params', 'transActorID'))
+	propAnnotations['transActorParamCustom'] = bpy.props.StringProperty(name = 'Actor Parameter', default = '0x0000')
+
+	# Other
+	# isActorSynced is used to check if the blend's data is from an older version of Fast64
+	propAnnotations['isActorSynced'] = bpy.props.BoolProperty(default=False)
```

Comment:
I think `isActorSynced` should be gotten rid of entirely, as I said it's duplicating what a version != cur_version check does,
and it's going to be useless besides this one particular version upgrade, it couldn't be reused for other version changes